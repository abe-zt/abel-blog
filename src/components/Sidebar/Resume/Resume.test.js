// // @flow strict
// import React from 'react';
// import renderer from 'react-test-renderer';
// import Resume from './Resume';

// describe('Resume', () => {
//   it('renders correctly', () => {
//     const props = {
//       resume: 'resume'
//     };

//     const tree = renderer.create(<Resume {...props} />).toJSON();
//     expect(tree).toMatchSnapshot();
//   });
// });
