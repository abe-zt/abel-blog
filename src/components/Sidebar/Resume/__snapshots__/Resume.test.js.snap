// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume renders correctly 1`] = `
<div
  className="resume"
>
  resume
</div>
`;
